### ISSUE TITLE
Pif 17.2 basic only

### ISSUE THREAD

**Issue by SunRizzze** [chiteroman/PlayIntegrityFix#404]
Only basic verdict![Screenshot_20240818-050827.png](https://github.com/user-attachments/assets/420a7305-a26b-46c4-bb4b-2d71310feb42)

**Comment by nvn152** [comment]
Same problem with 17.2

**Comment by alekyiv** [comment]
Basic only with 17.2 and tc 1.1.0
17.1 2/3
Los 20 @coduxer
Redmi note 11
Magisk 27006 canary
Zygisk 
Magisk Hide + random name package 
Keyatestation blue. + Bootloader spoof locked 
Rollback to 17.1

**Comment by PL7963** [comment]
Same with Xiaomi 13 Pro

**Comment by tajnymag** [comment]
Can confirm on hotdogb YAAP 14, which shouldn't be signed with test keys. After I downgraded to 17.1, I got DEVICE again. Something is off with 17.2.

**Comment by redhyper9** [comment]
Same here. I pass only basic integrity in both YASNAC and Play Integrity API Checker after upgrading to 17.2, with 17.1 there were no issues (PixelOS on OnePlus Nord CE2 Lite). Downgrading to 17.1 for now.

**Comment by ElDavoo** [comment]
Same with my problematic device (Poco x3 NFC with broken tee, with A13 NitrogenOS, which has no selinux, no seccomp, userdebug, test signed...) I can pass DEVICE with v17.1 but v17.2 gets me nothing (not even basic).

**Comment by bamzero** [comment]
ArrowOS 12 on Poco X3 Pro only basic, have tried enabling all spoofing options in pif.json but no luck. Downgraded to 17.1 with spoof package manager enabled for device again.

**Comment by chiteroman** [comment]
Zero issues on my 2 devices using stock ROM...

**Comment by bamzero** [comment]
> Zero issues on my 2 devices using stock ROM...

Seems issues are limited to custom ROMs then? Not sure what changes were made to Fix custom ROMs support as per the changelog but noticed no more spoof_package_manager in the json which I had previously been enabling. Just reverted a Redmi Note 9 Pro on Arrow OS 12 to v17.1 and left it disabled but still passing device though.

Well at least we've still got 17.1 for now, thanks for your continuing efforts regardless.

**Comment by Kuszki** [comment]
I can confirm (Evolution X 7.9.9 A13.0_r82 unsigned, Mi 9T):
Basic using 17.2 
Device using 17.1

BTW:
"SPOOF_PACKAGE_MANAGER" value in /data/adb/modules/playintegrityfix/pif.json is set to "false" by default and "Device" is achieved without changing it - this occurs since v16.9. Changing this value with 17.2 it doesn't influence verdict.

**Comment by reiryuki** [comment]
Same case in Redmi 4A (rolex) LineageOS Android 14
![Screenshot_20240818-201408_TB Checker](https://github.com/user-attachments/assets/9cd5ce92-83c1-4dc6-92e5-42d3348ca17c)
![Screenshot_20240818-201512_TB Checker](https://github.com/user-attachments/assets/f0a5f34a-1ee2-4293-adc1-cf9ed2821991)

**Comment by pv979** [comment]
Same here : only basic with 17.2
Matrixx ROM on marble (Poco f5)
Downgraded back to 17.1 for device integrity

**Comment by Renda80** [comment]
I have the same problem with the new version 17.2, there is only BASIC certification. When downgrading to version 17.1, the certification is DEVICE. I use Custom ROM with TestKey, with broken TTE, I don't use playcurl, or custom PIF.

**Comment by Renda80** [comment]
> I have the same problem with the new version 17.2, there is only BASIC certification. When downgrading to version 17.1, the certification is DEVICE. I use Custom ROM with TestKey, with broken TTE, I don't use playcurl, or custom PIF.

I tested the test build "PlayIntegrityFix_v17.3-TEST.zip" with DEVICE certification result. Version 17.2, there is only BASIC certification result. When downgrading to version 17.1, is DEVICE certification result. I use Custom ROM with TestKey, with broken TTE, I don't use Tricky-Store, playcurl, or custom PIF.

**Comment by foundObjects** [comment]
Device + Basic passing here on OnePlus LE2125, stock OOS 13.1.0.593, PIF 17.2, Magisk 26.3

![Screenshot_2024-08-19-09-50-18-47_0ecf870e1d5390b9ac21ebc76f00749d.jpg](https://github.com/user-attachments/assets/1e38a56d-375a-4479-8cdd-12ba071f9b3c)

Also passing Device+Basic on Samsung SM-T860 running stock 13.1, Magisk 27.0 and PIF 17.2

**Comment by tarockx** [comment]
Same here, latest update breaks integrity on latest AOSPA Uvite running on POCO F5 (marble).
Reverting to 17.1 fixes it for now, hope newer versions can be fixed as well.

**Comment by paresh996** [comment]
Flashed unreleased v17.3 off of the latest commit, passes basic and device again!

**Comment by Renda80** [comment]
> Flashed unreleased v17.3 off of the latest commit, passes basic and device again!

Yes, this is already known, it was already mentioned here in this post [.](https://github.com/chiteroman/PlayIntegrityFix/issues/404#issuecomment-2296881437)

**Comment by Renda80** [comment]
> > I have the same problem with the new version 17.2, there is only BASIC certification. When downgrading to version 17.1, the certification is DEVICE. I use Custom ROM with TestKey, with broken TTE, I don't use playcurl, or custom PIF.
> 
> I tested the test build "PlayIntegrityFix_v17.3-TEST.zip" with DEVICE certification result. Version 17.2, there is only BASIC certification result. When downgrading to version 17.1, is DEVICE certification result. I use Custom ROM with TestKey, with broken TTE, I don't use Tricky-Store, playcurl, or custom PIF.

I tested another test build "PlayIntegrityFix_v17.3-TEST2.zip" with DEVICE certification result. I use Custom ROM A11 with TestKey, with broken TTE, I don't use Tricky-Store, playcurl, or custom PIF.

**Comment by redhyper9** [comment]
Flashed the newly release v17.3 and now both BASIC & DEVICE are passing again. (PixelOS on OnePlus Nord CE2 Lite)

**Comment by tarockx** [comment]
I can confirm with v17.3 device integrity is back 🥰

(tested on marble with latest official AOSPA)

**Comment by ElDavoo** [comment]
Yeah @SunRizzze please close
